Ginger/GingerCoreCommon/VariablesLib/VariableNumber.cs

```diff
@@ -0,0 +1,242 @@
+﻿#region License
+/*
+Copyright © 2014-2020 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Enums;
+using Amdocs.Ginger.Repository;
+
+namespace GingerCore.Variables
+{
+    public class VariableNumber : VariableBase
+    {
+        private string mInitialNumberValue;
+
+        private float? mMinValue;
+        private float? mMaxValue;
+
+        private int? mPrecisionValue;
+
+        private bool mIsDecimalValue;
+        
+        [IsSerializedForLocalRepository]
+        public bool IsDecimalValue
+        {
+            set
+            {
+                mIsDecimalValue = value;
+                OnPropertyChanged("IsDecimalValue");
+                OnPropertyChanged("Formula");
+            }
+            get
+            {
+                return mIsDecimalValue;
+            }
+        }
+
+        [IsSerializedForLocalRepository]
+        public float? MinValue
+        {
+            get 
+            {
+                return mMinValue; 
+            }
+            set 
+            {
+                mMinValue = value;
+                OnPropertyChanged("MinValue");
+            }
+        }
+
+        [IsSerializedForLocalRepository]
+        public float? MaxValue
+        {
+            get 
+            { 
+                return mMaxValue;
+            }
+            set
+            { 
+                mMaxValue = value;
+                OnPropertyChanged("MaxValue");
+            }
+        }
+
+        public VariableNumber() { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5355677)